crates/lang/codegen/src/generator/metadata.rs

```diff
@@ -270,7 +275,7 @@ impl Metadata<'_> {
         match ret_ty {
             None => {
                 quote! {
-                    ::ink_metadata::ReturnTypeSpec::new(None)
+                    ::ink_metadata::ReturnTypeSpec::new(::core::option::Option::None)
```

Comment:
I feel like we should be able to do all these path clean-ups with a Bash script

Reply:
I hope we won't be trusting a bash script for transformations like these on our code. :sweat_smile: